cmake/Install.cmake

```diff
@@ -766,6 +766,10 @@ cpack_ifw_configure_component(Symlinks SCRIPT cmake/qtifw/install_unix_createsym
 
 cpack_ifw_configure_component(CreateStartMenu SCRIPT cmake/qtifw/install_win_createstartmenu.qs)
 
+if (PYTHON_CLI)
+  cpack_ifw_configure_component(Unspecified SCRIPT cmake/qtifw/install_auxiliary_python_shortcuts.qs)
+endif()
```

Comment:
Alright, so here is a new QtIFW script that is attached to the Unspecified component.  Could consider adding it to the Auxiliary or something if we wanted all the Python CLI app stuff in Auxiliary.  

Anyway, this script will handle any desktop integration steps to enhance the user experience.